DQM/Integration/python/clients/beamhlt_dqm_sourceclient-live_cfg.py

```diff
@@ -134,7 +134,7 @@
 
 # Set rawDataRepacker (HI and live) or hltFEDSelectorTCDS+hltFEDSelectorOnlineMetaData (for all the rest)
 if (process.runType.getRunType() == process.runType.hi_run and live):
-    rawDataInputTag = "rawDataRepacker"
+    rawDataInputTag = "hltFEDSelectorTCDS"
```

Comment:
see comment in master https://github.com/cms-sw/cmssw/pull/40109/files#r1026796774